java/ql/src/Language Abuse/EmptyMethod.md

```diff
@@ -0,0 +1,36 @@
+## Overview
+
+An empty method may indicate that an implementation was intended to be provided but was accidentally omitted. When using the method, it will not be clear that it does not provide an implementation and with dynamic dispatch, resolving to a blank method may result in unexpected program behaviour.
+
+## Recommendation
+
+If a method is intended to be left empty, do one of the following to indicate that it is intentionally empty: 1) mark it abstract in an abstract class, 2) place it in an interface (then it can be implicitly abstract), 3) place a comment in that method that lets others know that the implementation was intentionally omitted, or 4) throw an `UnsupportedOperationException` in the method (like is done in `java.util.Collection.add`).
```

Comment:
Would it make sense to have these options in a bulleted or numbered list, to improve readability?
Also fixed typo, and tweaked some content.

```suggestion
If a method is intended to be left empty, do one of the following to indicate that it is intentionally empty: 
1. Mark it abstract in an abstract class
2. Place it in an interface (then it can be implicitly abstract)
3. Place a comment in that method that lets others know that the implementation was intentionally omitted
4. Used `UnsupportedOperationException` in the method (as in `java.util.Collection.add`).
```